lib/faucet_pipeline_rails/railtie.rb

```diff
@@ -8,5 +8,19 @@ class Railtie < Rails::Railtie
         Manifest.instance.fetch(source)
       end
     end
+
+    initializer "faucet_pipeline.configure_manifest_path" do |app|
+      config.faucet_pipeline = ActiveSupport::OrderedOptions.new
+      config.faucet_pipeline.manifest_path = app.root.join("public", "assets", "manifest.json")
+    end
+
+    rake_tasks do
+      namespace :assets do
+        desc "Compile the assets via faucet-pipeline"
+        task :precompile do
+          sh "./node_modules/.bin/faucet --compact --fingerprint"
```

Comment:
Is `node_modules` at root level a valid assumption for Rails projects?

Reply:
I think it is, because that's the case when you just set it up as basic as possible. If you do more sophisticated things, you probably will also not use the precompile task. This is a help-thing for people coming from the Rails asset pipeline to ease them into this.

Reply:
I added a sentence to explain it.